benchmark/harness_results.go

```diff
@@ -43,33 +48,33 @@ func (r *BenchResult) EvergreenPerfFormat() ([]interface{}, error) {
 
 	out := []interface{}{
 		map[string]interface{}{
-			"name": r.Name + "-throughput",
-			"results": map[string]interface{}{
-				"1": map[string]interface{}{
-					"seconds":        r.Duration.Round(time.Millisecond).Seconds(),
-					"ops_per_second": r.getThroughput(median),
-					"ops_per_second_values": []float64{
-						r.getThroughput(min),
-						r.getThroughput(max),
-					},
+			"info": map[string]interface{}{
+				"test_name": r.Name + "-throughput",
+				"args": map[string]interface{}{
+					"threads": 1,
 				},
 			},
+			"metrics": []Metric{
+				{Name: "seconds", Value: r.Duration.Round(time.Millisecond).Seconds()},
+				{Name: "ops_per_second", Value: r.getThroughput(median)},
+				{Name: "ops_per_second_values", Value: []float64{r.getThroughput(min), r.getThroughput(max)}},
+			},
 		},
 	}
 
 	if r.DataSize > 0 {
 		out = append(out, interface{}(map[string]interface{}{
-			"name": r.Name + "-MB-adjusted",
-			"results": map[string]interface{}{
-				"1": map[string]interface{}{
-					"seconds":        r.Duration.Round(time.Millisecond).Seconds(),
-					"ops_per_second": r.adjustResults(median),
-					"ops_per_second_values": []float64{
-						r.adjustResults(min),
-						r.adjustResults(max),
-					},
+			"info": map[string]interface{}{
+				"test_name": r.Name + "-MB-adjusted",
+				"args": map[string]interface{}{
+					"threads": 1,
 				},
 			},
+			"metrics": []Metric{
+				{Name: "seconds", Value: r.Duration.Round(time.Millisecond).Seconds()},
+				{Name: "ops_per_second", Value: r.adjustResults(median)},
+				{Name: "ops_per_second_values", Value: []float64{r.adjustResults(min), r.adjustResults(max)}},
```

Comment:
Similar suggestion of splitting this into two metrics: `ops_per_second_min` and `ops_per_second_max`.